.gitignore

```diff
@@ -23,3 +24,4 @@ dev/
 lib/i18next/
 lib/jquery-i18next/
 lib/magnific-popup/
+.idea
```

Comment:
I used the Idea IntelliJ IDE to code the feature (just to see how it works with plain Javascript projects). This IDE create a hidden directory (.idea here) to store the project settings.